pkg/northbound/admin/admin.go

```diff
@@ -21,6 +21,7 @@ import (
 	"github.com/onosproject/onos-config/pkg/manager"
 	"github.com/onosproject/onos-config/pkg/northbound"
 	"github.com/onosproject/onos-config/pkg/store"
+	networkchange "github.com/onosproject/onos-config/pkg/types/change/network"
```

Comment:
alias as `networkchangetypes` to be consistent

Reply:
done